src/amo/components/CollectionList/index.js

```diff
@@ -0,0 +1,179 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import {
+  expandCollections,
+  fetchUserCollections,
+} from 'amo/reducers/collections';
+import { getCurrentUser } from 'amo/reducers/users';
+import AuthenticateButton from 'core/components/AuthenticateButton';
+import { withFixedErrorHandler } from 'core/errorHandler';
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import CardList from 'ui/components/CardList';
+import UserCollection from 'ui/components/UserCollection';
+import type {
+  CollectionsState,
+  CollectionType,
+} from 'amo/reducers/collections';
+import type { UsersStateType } from 'amo/reducers/users';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+import type { ReactRouterLocation } from 'core/types/router';
+
+import './styles.scss';
+
+export type Props = {||};
+
+export type InternalProps = {|
+  ...Props,
```

Comment:
Because it is a "page-level component", I think we could just use `Props` here, no need for an empty type `Props` and `InternalProps`.